continuousflex/protocols/protocol_subtomograms_synthesize.py

```diff
@@ -581,6 +582,27 @@ def generate_rotation_and_shift(self):
             else:
                 psi1 = np.random.normal(self.MeanPsi.get(), self.StdPsi.get())
 
+            # uniform over the sphere
```

Comment:
This make me actually wonder if we need to keep the "Gaussian distribution" options at all. It seems to be complicating things and we have never used it.

Reply:
That is indeed much easier with scipy! 
Yes... I believe it is useless we should remove it.
I am not sure that choosing the range of the angles is useful as well, I never changed this range, maybe we should just give the possibility to set the range of translation and for rotation just set yes or no you want to synthesize the rotations

Reply:
I agree. Shifts make sense to be simulated in Gaussian distribution around zero since the person (or algorithm) tries to hit the center. But angles have no meaning like that. There are a couple of complex patterns that I have noticed in preferred orientations, I will think how to implement that later (probably an interactive tool).